Source/Controllers/SPAppController.m

```diff
@@ -247,28 +245,6 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
         }
     }
 
-    executeOnBackgroundThread(^{
-        NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];
-        FIRCrashlytics *crashlytics = FIRCrashlytics.crashlytics;
-
-        // fake the dbViewInfoPanelSplit being open
-        NSMutableArray *dbViewInfoPanelSplit = [[NSMutableArray alloc] initWithCapacity:2];
-        [dbViewInfoPanelSplit addObject:@"0.000000, 0.000000, 359.500000, 577.500000, NO, NO"];
-        [dbViewInfoPanelSplit addObject:@"0.000000, 586.500000, 359.500000, 190.500000, NO, NO"];
-        [prefs setObject:dbViewInfoPanelSplit forKey:@"NSSplitView Subview Frames DbViewInfoPanelSplit"];
```

Comment:
@Kaspik Was this an intentional remove? I think these lines are unrelated to crashlytics

Reply:
Reverted

Reply:
No we need that. That was the 'fix' for the eye-information going whacky bug.

Reply:
Looks like @Kaspik reverted these lines back in main. Resolving convo! :)